### ISSUE TITLE
Don't display "archived (metadata only)" for unavailable page archives

### ISSUE THREAD

**Issue by spiriiii** [TheTechRobo/youtubevideofinder#63]
Many videos display a metadata only link even though the page it links to says "Video unavailable"
Not sure if this can be fixed though

**Comment by TheTechRobo** [comment]
Yeah, not sure how to consistently fix this. #62 would hopefully improve that, though. 

Even a check for unavailable videos wouldn't be 100% correct, as I have seen a video where even on the "video unavailable" page it contains the video metadata. Maybe it should check for the video metadata, but there are so many versions of the watch page that I don't know how well that would work... :/
